app-editors/gvim/gvim-8.2.4586.ebuild

```diff
@@ -0,0 +1,383 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# Please bump with app-editors/vim-core and app-editors/gvim
+
+VIM_VERSION="8.2"
+LUA_COMPAT=( lua5-1 luajit )
+PYTHON_COMPAT=( python3_{7..10} )
+PYTHON_REQ_USE="threads(+)"
+USE_RUBY="ruby24 ruby25 ruby26 ruby27"
+
+inherit vim-doc flag-o-matic xdg-utils bash-completion-r1 prefix lua-single python-single-r1 ruby-single
+
+if [[ ${PV} == 9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/vim/vim.git"
+	EGIT_CHECKOUT_DIR=${WORKDIR}/vim-${PV}
+else
+	SRC_URI="https://github.com/vim/vim/archive/v${PV}.tar.gz -> vim-${PV}.tar.gz
+		https://dev.gentoo.org/~zlogene/distfiles/app-editors/vim/vim-8.2.0360-gentoo-patches.tar.xz"
+	KEYWORDS="~alpha amd64 ~arm ~arm64 ~hppa ~ia64 ppc ppc64 ~riscv ~s390 sparc x86 ~amd64-linux ~x86-linux ~ppc-macos ~x86-solaris"
```

Comment:
Please use `ekeyword \~all foo.ebuild` for bumps (see [devmanual](https://devmanual.gentoo.org/keywording/index.html#keywording-on-upgrades))

Reply:
Marked all arch with ~